extensions/github-authentication/media/auth.css

```diff
@@ -33,90 +33,84 @@ body {
 	display: flex;
 	flex-direction: column;
 	color: var(--vscode-foreground);
-	font-family: "Segoe UI","Helvetica Neue","Helvetica",Arial,sans-serif;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", system-ui, "Ubuntu", "Droid Sans", sans-serif;
```

Comment:
this copies our VS Code fonts with fallbacks per platform, I did not understand why we had the font faces with external URLs, this feels better to me and matches what vscode does.